internal/webhookhandler/webhookhandler.go

```diff
@@ -35,6 +35,8 @@ import (
 // +kubebuilder:rbac:groups=opentelemetry.io,resources=instrumentations,verbs=get;list;watch
 // +kubebuilder:rbac:groups="apps",resources=replicasets,verbs=get;list;watch
 
+const initContainerName = "opentelemetry-auto-instrumentation"
```

Comment:
This package should not know anything about the consumers (one of which is auto-instrumentation pod mutator). 

there are a couple of possible solutions 
* move the code to the auto-instrumentation pod mutator
* add a new method to the pod mutator interface
* create a check func/type and pass it to the webhook factory method

Reply:
I've moved code to `instrumentation`.